MediaWikiKit/MediaWikiKit/MWKSavedPageList.m

```diff
@@ -35,14 +43,18 @@ - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     return self;
 }
 
-#pragma mark - Entry Access
-
-- (MWKSavedPageEntry*)entryAtIndex:(NSUInteger)index {
-    return [super entryAtIndex:index];
+- (void)importEntries:(NSArray*)entries {
+    NSArray<MWKSavedPageEntry*>* validEntries = [entries bk_reject:^BOOL (MWKSavedPageEntry* entry) {
+        return entry.title.text.length == 0;
+    }];
+    NSArray<MWKSavedPageEntry*>* uniqueValidEntries = [[NSOrderedSet orderedSetWithArray:validEntries] array];
```

Comment:
i'm pretty sure it would keep the one already in the list. it's probably an edge case and i'm not sure it matters which one we keep, or which one we'd _want_ to keep. probably whichever one had a newer date and/or not-unknown discovery method. i saw we can implement custom-dedupe logic if/when we get a bug report